### PR TITLE
Drop Python 2

### PR DESCRIPTION
**Pull request by niboshi**

### CODE REVIEW

**Comment by kmaehashi** [comment]
pfnCI, test this please.

**Comment by niboshi** [comment]
Fixed.

**Comment by kmaehashi** [comment]
pfnCI, test this please.

File: run_cupy_install_test.py [hunk 1]
```diff
@@ -39,7 +39,7 @@
     # make sdist
     # cuda, cudnn and numpy is required to make a sdist file.
     build_conf = {
-        'base': 'ubuntu16_py27',
+        'base': 'ubuntu18_py38-pyenv',
```
Comment (kmaehashi) [inline run_cupy_install_test.py]: CUDA 8.0 does not support Ubuntu 18.
```suggestion
        'base': 'ubuntu16_py35',
```

c.f. https://github.com/chainer/chainer-test/blob/fd2f22a6989a9567c12b2683ff9e8cd0b46a502f/shuffle.py#L173

Comment (niboshi) [reply run_cupy_install_test.py]: Thanks, fixed in 0c429688a6f3c49a64271b062d387ce0f0badb76.

**Comment by kmaehashi** [comment]
pfnCI, test this please.

**Comment by niboshi** [comment]
Conflict resolved.

**Comment by kmaehashi** [comment]
pfnCI, test this please.

**Pull request merged**